Cargo.toml

```diff
@@ -12,9 +12,10 @@ keywords = ["kyber", "kem", "key-exchange", "kex", "post-quantum"]
 readme = "readme.md"
 
 [dependencies]
-rand_core = {version = "0.6.2",  default-features = false }
-wasm-bindgen = { version = "0.2.74", optional = true }
-sha2 = { version = "0.9.5", optional = true }
+rand_core = { version = "0.6", default-features = false }
+wasm-bindgen = { version = "0.2", optional = true }
```

Comment:
Is removing the patch version the part about being "sane"? I'd argue that specifying the lower bound on the patch is very sane and healthy. Well, depending on how well you test your code with the *oldest allowed version* of course. It could very well be that this library relies on a feature added to `rand_core 0.6.1`, such a release can introduce features, but not break any APIs. Unless explicitly tested with the oldest versions of all dependencies, not specifying a lower bound on the patch version is bad practice IMO.

https://github.com/mullvad/coding-guidelines/blob/master/rust.md#specifying-version-numbers

Reply:
The fact is unless this crate is actively maintained, having such pinned dependencies will just make user not be able to bump to newer subversion when patch comes in (and for such big crates as rand_core and wasm-bindgen it's pretty much a guarantee that you will have dependencies breakage with larger project that use multiple dependencies)

Reply:
Rand_core is now 0.6.3 since june 15th 2021: https://docs.rs/rand_core/0.6.3/rand_core/
Wasm bindgen is now 0.2.82 since 22 days ago (0.2.74 is now 1 years and 3 months old):  https://docs.rs/wasm-bindgen/0.2.82/wasm_bindgen/
Sha2 is now 0.10.2 which added a new stabilization in Rust crypto trait (very much needed)

Reply:
How is the level of maintenance relevant? If no one ever runs this crate against `cargo +nightly update -Z minimal-versions && cargo check` developers will very likely accidentally miss that they start relying on features not available in the oldest allowed versions of their dependencies. I usually prefer to have the above check in my CI, for safety.

> having such pinned dependencies will just make user not be able to bump to newer subversion when patch comes in

I don't think I understand what you mean here. How is setting a **lower bound** on the patch version going to stop anyone from upgrading anything? Please note that the specified version is only a lower bound. No one suggested pinning an exact version of any dependency (yes, that is indeed bad practice, I agree!)

Reply:
In case this misunderstanding is about what I suspect it is about, please compare [Caret requirements](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements) with [Comparison requirements](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#comparison-requirements). This crate currently does not use `=x.y.z`, and thus, it does not pin any dependency to an exact version. It only sets a lower bound.

Reply:
This was fixed